num256/src/lib.rs

```diff
@@ -250,6 +290,25 @@ impl CheckedMul for Int256 {
   }
 }
 
+impl Div for Int256 {
+  type Output = Int256;
+  fn div(self, v: Int256) -> Int256 {
+    let num = self.0 / v.0;
```

Comment:
https://github.com/rust-lang/rfcs/issues/1368
A quote from the issue above:
fwiw, I see no mention here of "thread-directed" signals. IMHO, it's reasonable to not support them as a first pass (they're a totally different thing than process-directed signals), but worth at least being explicit about what's in scope.

As an example of what these are: let's say I do a mmap of a file and try to read from it but another process has truncated the file so the page I'm reading is no longer valid. A SIGBUS is generated and directed to the thread doing the read. It can't be handled by some other thread using signalfd or the like.

Likewise, SIGFPE, SIGSEGV, SIGILL, SIGPIPE, and anything done with pthread_kill/pthread_sigqueue. The last case is sometimes used as a mechanism for CPU / wall clock profiling tools or thread cancellation.

Reply:
This is a bigint, so I think it will panic instead of raising a hardware exception, but I have added div by zero checker